### DISCUSSION TITLE
Use of Torchmetrics

### DISCUSSION THREAD

**Discussion by RaivoKoot** [pykale/pykale#280] [Ideas] [answered]
This will be the central discussion for the use of [TorchMetrics](https://torchmetrics.readthedocs.io/en/latest/) instead of implementing them ourselves into pykale.

An example is [top-k accuracy](https://torchmetrics.readthedocs.io/en/latest/references/modules.html#id4) and PR #273

**Comment by xianyuanliu** [comment, marked as answer]
Integrated into #402.
